code/game/area/area.dm

```diff
@@ -59,8 +59,12 @@
 	var/min_ambience_cooldown = 40 SECONDS
 	///Used to decide what the maximum time between ambience is
 	var/max_ambience_cooldown = 120 SECONDS
-
-
+	///Assoc list of faction members in this area; only used in civil war to determine who controls this zone
+	var/list/alive_faction_member_in_area
+	///Who controls this area
+	var/faction_controlling = ""
```

Comment:
why is this an empty string instead of null

Reply:
to make it easier to understand that it's a string